src/constants/dexes.ts

```diff
@@ -277,6 +277,10 @@ export const dexListConfig: DexList = {
     name: 'Velodrome',
     icon: 'https://s2.coinmarketcap.com/static/img/coins/64x64/20435.png',
   },
+  dystopia: {
+    name: 'Dystopia',
+    icon: 'https://storage.googleapis.com/ks-setting-1d682dca/7693d113-5321-414c-a7c7-11d501b49379.png',
+  },
```

Comment:
No need

Reply:
You can refactor to remove this file, or most line of this file

Reply:
Ok. I removed it